setendings.py

```diff
@@ -0,0 +1,397 @@
+"""Helper module for ending_location_visitor.py
+"""
+from sys import exit
+import astroid
+import logging
+
+# Set the log level (DEBUG, ERROR, ...), and message format.
+logging.basicConfig(format='', level=logging.DEBUG)
+
+# Helper functions to abstract common tasks. Global for now..
+# these functions are associated to types in the transforms dictionary in the
+# TransformVisitor class. The _transform method invokes the transform function
+# on the nodes.
+# TODO: Separate into Class? .. would need static methods.
+################################################################################
+
+def _set_end_lineno(node):
+    """Transform the property in-place.
+    Testing with simple, single-line things for now.
+    Goal is to work on various multi-line nodes, with complicated nesting.
+    'tolineno' is set correctly on everything, by asteroid.
+    """
+    # Guard. 
+    if node.tolineno is None:
+        # TODO: write new code to gracefully handle, if this is ever reached.
+        raise Exception('''ERROR:️ lineno is None at node:''', node, 
+                        node.as_string())
+    else:
+        node.end_lineno = node.tolineno
+
+def _set_end_col_offset_by_string(node):
+    """Nodes without children can get end_col_offset by length of string.
+    Hopefully this works without problems.
+    """
+    NO_CHILDREN_TYPE = [astroid.Arguments, astroid.AssignName, astroid.Break, 
```

Comment:
You can move this into the global scope. I imagine eventually (but not now) you'll want to call `register_transform` on each of these. Although I do wonder about some of these, and would want to investigate further before doing so.